src/foam/nanos/crunch/ui/CapabilityWizardlet.js

```diff
@@ -50,8 +51,24 @@ foam.CLASS({
       name: 'title',
       class: 'String',
       expression: function(capability) {
-        if ( ! capability || ! capability.name ) return '';
-        return capability.name;
+        var expr = foam.mlang.Expressions.create();
```

Comment:
Easier to do implements: [ ''foam.mlang.Expressions' ], then just do this. instead of expr.

Reply:
fixed